test/features/can_revert_record_test.rb

```diff
@@ -22,15 +22,16 @@ class CanRevertRecordTest < Capybara::Rails::TestCase
       visit '/home'
 
       within '#in_daac_review' do
-        check 'record_id_'
+        page.save_and_open_page
+        all('#record_id_')[0].click
```

Comment:
rest of just updates are test classes to account for the fact that some of the test data doesn't have a version data and since I fixed the code to pull all collection records, not just collection records with a version, we are seeing a couple extra rows.  So the test data is just grabbing the first which happens to be the correct one.